validate/error.go

```diff
@@ -0,0 +1,69 @@
+package validate
```

Comment:
> But OCIErrorCode can not be shared between runtime-tool and image-tool. I think we can keep it here.

Yeah.  A stand-alone module with all the tooling and a public code-> string map, and then the validating consumer populates that map (and spec URL) for whatever it's testing.